examples/api/demo_affine_image.py

```diff
@@ -24,22 +21,15 @@ def get_image():
     return Z
 
 
-def imshow_affine(ax, z, *kl, **kwargs):
-    im = ax.imshow(z, *kl, **kwargs)
-    x1, x2, y1, y2 = im.get_extent()
-    im._image_skew_coordinate = (x2, y1)
```

Comment:
Yes, it does.  Not much we can do except stop doing that going forward.  On the up side, the experimental method is also completely redundant to the more flexible and consistent-with-everything-else transforms infrastructure, so this isn't a regression in terms of available functionality.